source/less/blocks/instruction.less

```
@@ -17,6 +21,38 @@
   margin: 0;
   padding: 0;
   list-style: none;
+
+  @media (min-width: @tablet-width) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+}
+
+.instruction__item {
+  &--1 {
+    @media (min-width: @tablet-width) {
+      width: 230px;
+    }
+  }
+
+  &--2 {
+    @media (min-width: @tablet-width) {
+      width: 246px;
+    }
+  }
+
+  &--3 {
+    @media (min-width: @tablet-width) {
+      width: 247px;
+    }
+  }
+
+  &--4 {
+    @media (min-width: @tablet-width) {
+      width: 249px;
+    }
+  }
 }
```

Слишком хрупкое решение. Если текст станет немного другим, всё может поломаться.
Не всегда стоит копировать значения из редактора.
Здесь ты можешь просто взять максимальный размер из всех блоков (250px) и использовать его, плюс одинаковые отступы:

```suggestion
.instruction__item {
  @media (min-width: @tablet-width) {
    width: 250px;
    margin-right: 104px;
    margin-bottom: 37px;
  }

  @media (min-width: @desktop-width) {
    margin-right: 0;
  }
}
```
А для десктопа отступы можно обнулить (хотя бы горизонтальные) и использовать для выравнивания flex-свойства родителя